Sources/CoreCommands/SwiftToolObservabilityHandler.swift

```diff
@@ -198,7 +198,8 @@ extension ObservabilitySystem {
     public static func swiftTool(
         outputStream: OutputByteStream = stdoutStream,
         logLevel: Basics.Diagnostic.Severity = .warning
-    ) -> ObservabilitySystem {
-        .init(SwiftToolObservabilityHandler(outputStream: stdoutStream, logLevel: logLevel))
+    ) -> (ObservabilitySystem, SwiftToolObservabilityHandler) {
```

Comment:
This change is needed to allow getting a reference to `SwiftToolObservabilityHandler` and calling `wait` on it.

Reply:
I think this is fine, but at some point we should consider if these new command should become aligned with the other commands so they dont need this kind of external access

Reply:
I found a better way to address that within this PR. Turns out this `static func swiftTool` was used just once in `DestinationCommand`, it made more sense to move its implementation over there, so there's no need for this tuple-returning function anymore.